### DISCUSSION TITLE
[ 潜水 ] 给音游上点难度吧！不过瘾呀！

### DISCUSSION THREAD

**Discussion by uuutt2023** [share121/inter-knot#3101] [其他]
![封面](https://vip.helloimg.com/i/2024/08/19/66c314c107f7e.png)
![截图1.png](https://vip.helloimg.com/i/2024/08/19/66c314bf4fb52.png)
![截图2.png](https://vip.helloimg.com/i/2024/08/19/66c314c026ec0.png)

**Comment by fish-phaethon** [comment]
建议每个不同的按键对应不同颜色的圈，还嫌难度低可以搞一个混合圈

**Comment by WhereisFff** [comment]
的确啊，太简单了不过瘾啊！

**Comment by Nie7bai** [comment]
非常简单但鼻青脸肿<img src="https://img-static.mihoyo.com/communityweb/upload/0f40e7b49c17f5575f3296cf36ab22ff.gif" alt="派蒙-惊吓" width="75" >

**Comment by MLx-x** [comment]
虽然我也觉得简单，但是放过那些赛博老登吧<img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" >

**Comment by I-m-Miss-Lucy-s-dog** [comment]
開放玩家自製譜就好玩了

**Reply by D111Z41SY** [reply]
活动的流程什么的，多迭代几次会更优秀，下次说不定就有玩家自制环节了。隔壁原神项目组的躲猫猫和音游小游戏都在逐渐改善呢（虽然还是一言难尽）

**Comment by D111Z41SY** [comment]
才第三天，后面可能会有难一点的吧。求放过，我只想拿了菲林就跑
